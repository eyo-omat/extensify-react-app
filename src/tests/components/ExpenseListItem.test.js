import React from 'react';
import {shallow} from 'enzyme';
import ExpenseListItem from '../../components/ExpenseListItem';
import expenseData from '../fixtures/expenses';

test('should render expenseListItem with expense data', () => {
    const wrapper = shallow(<ExpenseListItem {...expenseData[1]}/>);
    expect(wrapper).toMatchSnapshot();

});


