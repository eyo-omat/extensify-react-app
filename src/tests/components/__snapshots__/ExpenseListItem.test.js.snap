// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render expenseListItem with expense data 1`] = `
<div>
  <Link
    replace={false}
    to="/edit/2"
  >
    <h3>
      Mac
    </h3>
  </Link>
  <p>
    450
     - 
    345600000
     
  </p>
</div>
`;
